cmd/calyptia/get_environment.go

```diff
@@ -63,17 +63,10 @@ func (config *config) loadEnvironmentID(environmentKey string) (string, error) {
 		return "", err
 	}
 
-	if len(aa.Items) != 1 && !validUUID(environmentKey) {
-		if len(aa.Items) != 0 {
-			return "", fmt.Errorf("ambiguous core instance name %q, use ID instead", environmentKey)
-		}
-
-		return "", fmt.Errorf("could not find core instance %q", environmentKey)
-	}
+	if len(aa.Items) == 0 {
+		return "", fmt.Errorf("could not find environment %q", environmentKey)
 
-	if len(aa.Items) == 1 {
-		return aa.Items[0].ID, nil
 	}
 
-	return environmentKey, nil
```

Comment:
Removing this statement it will cause to break if the client inform an ID, once that we r returning an error when `if len(aa.Items) == 0` so we never use the ID after it. With this deleted code, in case we dont find by name we check if `environmentKey` its a valid ID and we return it

Reply:
I am not sure if following. In which case this function will get an ID?, why should we support ID(s) rather than names if names are unique among a project?